laborator/content/linking/demo/extern/extern-3.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+extern int var;
+int main(void)
+{
+	var = 10;
```

Comment:
Ar trebui sa scrii un comentariu in care explici de ce nu linkeaza.

Reply:
Asta e smecheria, vreau sa am si unele erori si sa se intreb: "De ce nu merge X? Ce am putea face sa mearga?"